simulation/simple_sensor_simulator/include/simple_sensor_simulator/sensor_simulation/traffic_lights/traffic_lights_detector.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 TIER IV, Inc. All rights reserved.
+// Copyright 2024 TIER IV, Inc. All rights reserved.
```

Comment:
there are some policy for the year description 
 in copyright notice, and we use the 2015(TIER IV founded year) for it throughtout in this repository.
 I know there is a way to update the year as changes occur, but it is difficult to keep up to date.
```suggestion
// Copyright 2015 TIER IV, Inc. All rights reserved.
```